### ISSUE TITLE
Server crashed

### ISSUE THREAD

**Issue by Giulizo** [pmmp/PocketMine-MP#3666]
Link to crashdump: https://pastebin.com/vu2S9kUs

### Additional comments (optional)

Everything worked great - I didn't add or remove any plugin, but the server suddenly crashed after 1-2 minutes every time you are in the world called "Freebuild" and flying around.

**Comment by SOF3** [comment]
```
{
    "format_version": 3,
    "time": 1593600769,
    "lastError": {
        "type": "InvalidStateException",
        "message": "Attempted to create tile pocketmine\\tile\\EnderChest in unloaded chunk 92 7",
        "fullFile": "phar:\/\/\/home\/minecraft\/multicraft\/servers\/server120687\/custom.phar\/src\/pocketmine\/level\/Level.php",
        "file": "src\/pocketmine\/level\/Level",
        "line": 2607,
        "trace": [
            "#0 src\/pocketmine\/tile\/Tile(171): pocketmine\\level\\Level->addTile(object pocketmine\\tile\\EnderChest)",
            "#1 src\/pocketmine\/tile\/Spawnable(62): pocketmine\\tile\\Tile->__construct(object pocketmine\\level\\Level, object pocketmine\\nbt\\tag\\CompoundTag)",
            "#2 src\/pocketmine\/tile\/Tile(117): pocketmine\\tile\\Spawnable->__construct(object pocketmine\\level\\Level, object pocketmine\\nbt\\tag\\CompoundTag)",
            "#3 src\/pocketmine\/level\/format\/Chunk(721): pocketmine\\tile\\Tile::createTile(string[10] EnderChest, object pocketmine\\level\\Level, object pocketmine\\nbt\\tag\\CompoundTag)",
            "#4 src\/pocketmine\/level\/Level(2681): pocketmine\\level\\format\\Chunk->initChunk(object pocketmine\\level\\Level)",
            "#5 src\/pocketmine\/level\/Level(2293): pocketmine\\level\\Level->loadChunk(integer 93, integer 8, boolean 1)",
            "#6 src\/pocketmine\/level\/Level(2955): pocketmine\\level\\Level->getChunk(integer 93, integer 8, boolean 1)",
            "#7 src\/pocketmine\/Player(1042): pocketmine\\level\\Level->populateChunk(integer 93, integer 8)",
            "#8 src\/pocketmine\/Player(1841): pocketmine\\Player->sendNextChunk()",
            "#9 src\/pocketmine\/Server(2401): pocketmine\\Player->checkNetwork()",
            "#10 src\/pocketmine\/Server(2154): pocketmine\\Server->tick()",
            "#11 src\/pocketmine\/Server(1994): pocketmine\\Server->tickProcessor()",
            "#12 src\/pocketmine\/Server(1588): pocketmine\\Server->start()",
            "#13 src\/pocketmine\/PocketMine(273): pocketmine\\Server->__construct(object BaseClassLoader, object pocketmine\\utils\\MainLogger, string[48] \/home\/minecraft\/multicraft\/servers\/server120687\/, string[56] \/home\/minecraft\/multicraft\/servers\/server120687\/plugins\/)",
            "#14 src\/pocketmine\/PocketMine(304): pocketmine\\server()",
            "#15 (11): require(string[96] phar:\/\/\/home\/minecraft\/multicraft\/servers\/server120687\/custom.phar\/src\/pocketmin)"
        ]
    },
    "error": {
        "type": "InvalidStateException",
        "message": "Attempted to create tile pocketmine\\tile\\EnderChest in unloaded chunk 92 7",
        "file": "src\/pocketmine\/level\/Level",
        "line": 2607
    },
    "plugin_involvement": "none",
    "code": {
        "2598": "\t\t\tthrow new LevelException(\"Invalid Tile world\");",
        "2599": "\t\t}",
        "2600": "",
        "2601": "\t\t$chunkX = $tile->getFloorX() >> 4;",
        "2602": "\t\t$chunkZ = $tile->getFloorZ() >> 4;",
        "2603": "",
        "2604": "\t\tif(isset($this->chunks[$hash = ((($chunkX) & 0xFFFFFFFF) << 32) | (( $chunkZ) & 0xFFFFFFFF)])){",
        "2605": "\t\t\t$this->chunks[$hash]->addTile($tile);",
        "2606": "\t\t}else{",
        "2607": "\t\t\tthrow new \\InvalidStateException(\"Attempted to create tile \" . get_class($tile) . \" in unloaded chunk $chunkX $chunkZ\");",
        "2608": "\t\t}",
        "2609": "",
        "2610": "\t\t$this->tiles[$tile->getId()] = $tile;",
        "2611": "\t\t$this->clearChunkCache($chunkX, $chunkZ);",
        "2612": "\t}",
        "2613": "",
        "2614": "\t\/**",
        "2615": "\t * @return void",
        "2616": "\t * @throws LevelException",
        "2617": "\t *\/"
    },
    "trace": [
        "#0 src\/pocketmine\/tile\/Tile(171): pocketmine\\level\\Level->addTile(object pocketmine\\tile\\EnderChest)",
        "#1 src\/pocketmine\/tile\/Spawnable(62): pocketmine\\tile\\Tile->__construct(object pocketmine\\level\\Level, object pocketmine\\nbt\\tag\\CompoundTag)",
        "#2 src\/pocketmine\/tile\/Tile(117): pocketmine\\tile\\Spawnable->__construct(object pocketmine\\level\\Level, object pocketmine\\nbt\\tag\\CompoundTag)",
        "#3 src\/pocketmine\/level\/format\/Chunk(721): pocketmine\\tile\\Tile::createTile(string[10] EnderChest, object pocketmine\\level\\Level, object pocketmine\\nbt\\tag\\CompoundTag)",
        "#4 src\/pocketmine\/level\/Level(2681): pocketmine\\level\\format\\Chunk->initChunk(object pocketmine\\level\\Level)",
        "#5 src\/pocketmine\/level\/Level(2293): pocketmine\\level\\Level->loadChunk(integer 93, integer 8, boolean 1)",
        "#6 src\/pocketmine\/level\/Level(2955): pocketmine\\level\\Level->getChunk(integer 93, integer 8, boolean 1)",
        "#7 src\/pocketmine\/Player(1042): pocketmine\\level\\Level->populateChunk(integer 93, integer 8)",
        "#8 src\/pocketmine\/Player(1841): pocketmine\\Player->sendNextChunk()",
        "#9 src\/pocketmine\/Server(2401): pocketmine\\Player->checkNetwork()",
        "#10 src\/pocketmine\/Server(2154): pocketmine\\Server->tick()",
        "#11 src\/pocketmine\/Server(1994): pocketmine\\Server->tickProcessor()",
        "#12 src\/pocketmine\/Server(1588): pocketmine\\Server->start()",
        "#13 src\/pocketmine\/PocketMine(273): pocketmine\\Server->__construct(object BaseClassLoader, object pocketmine\\utils\\MainLogger, string[48] \/home\/minecraft\/multicraft\/servers\/server120687\/, string[56] \/home\/minecraft\/multicraft\/servers\/server120687\/plugins\/)",
        "#14 src\/pocketmine\/PocketMine(304): pocketmine\\server()",
        "#15 (11): require(string[96] phar:\/\/\/home\/minecraft\/multicraft\/servers\/server120687\/custom.phar\/src\/pocketmin)"
    ],
    "general": {
        "name": "PocketMine-MP",
        "base_version": "3.14.0",
        "build": 1809,
        "is_dev": false,
        "protocol": 407,
        "git": "461bc94236ba1f3504b6f5a20eba9df80ef9a7bd",
        "uname": "Linux df4ad90b8691 4.19.62-mod-std-ipv6-64-rescue #828825 SMP Tue Jul 30 13:54:49 UTC 2019 x86_64",
        "php": "7.3.14",
        "zend": "3.3.14",
        "php_os": "Linux",
        "os": "linux",
        "composer_libraries": {
            "adhocore\/json-comment": "0.1.0@8448076039389f558f39ad0553aab87db3f81614",
            "ocramius\/package-versions": "1.5.1@1d32342b8c1eb27353c8887c366147b4c2da673c",
            "pocketmine\/binaryutils": "0.1.11@e8cb65db1b7998eebb739b124f2a989fe87366eb",
            "pocketmine\/callback-validator": "1.0.1@4aef6bb25f97d0b830ba0b4f66834b1a9c86fc9a",
            "pocketmine\/classloader": "0.1.0@4ccdb30e48f030bfcad04bb0a208d198ec631993",
            "pocketmine\/log": "0.2.0@e59bedb5d4bbeb9a26647cb7c367cb2fa72addfa",
            "pocketmine\/log-pthreads": "0.1.0@9bbcef398b01487ab47c234a6a7054722abbe067",
            "pocketmine\/math": "0.2.4@b1c28b236df8b795d7b06cf8421f9962b12ac410",
            "pocketmine\/nbt": "0.2.14@b31dca98443328c4bab76dc1b84c1473c79be20e",
            "pocketmine\/raklib": "0.12.8@9fea763ffc922db083e4528d4acd0b13d0ea83a4",
            "pocketmine\/snooze": "0.1.2@88420da3d9335dbcb3ee2decfd5e5453d057dcdf",
            "pocketmine\/spl": "0.4.1@ff0579a0be41bbe65d3637607715c0f87728a838",
            "doctrine\/instantiator": "1.3.1@f350df0268e904597e3bd9c4685c53e0e333feea",
            "myclabs\/deep-copy": "1.9.5@b2c28789e80a97badd14145fda39b545d83ca3ef",
            "phar-io\/manifest": "1.0.3@7761fcacf03b4d4f16e7ccb606d4879ca431fcf4",
            "phar-io\/version": "2.0.1@45a2ec53a73c70ce41d55cedef9063630abaf1b6",
            "phpdocumentor\/reflection-common": "2.1.0@6568f4687e5b41b054365f9ae03fcb1ed5f2069b",
            "phpdocumentor\/reflection-docblock": "5.1.0@cd72d394ca794d3466a3b2fc09d5a6c1dc86b47e",
            "phpdocumentor\/type-resolver": "1.2.0@30441f2752e493c639526b215ed81d54f369d693",
            "phpspec\/prophecy": "v1.10.3@451c3cd1418cf640de218914901e51b064abb093",
            "phpstan\/phpstan": "0.12.29@9771daaf6b95c6313b908d0bcdee0afcd51f838a",
            "phpstan\/phpstan-phpunit": "0.12.11@ab783a8ea634ea23305a8818c4750603e714489b",
            "phpstan\/phpstan-strict-rules": "0.12.2@a670a59aff7cf96f75d21b974860ada10e25b2ee",
            "phpunit\/php-code-coverage": "8.0.2@ca6647ffddd2add025ab3f21644a441d7c146cdc",
            "phpunit\/php-file-iterator": "3.0.2@eba15e538f2bb3fe018b7bbb47d2fe32d404bfd2",
            "phpunit\/php-invoker": "3.0.1@62f696ad0d140e0e513e69eaafdebb674d622b4c",
            "phpunit\/php-text-template": "2.0.1@0c69cbf965d5317ba33f24a352539f354a25db09",
            "phpunit\/php-timer": "5.0.0@b0d089de001ba60ffa3be36b23e1b8150d072238",
            "phpunit\/php-token-stream": "4.0.2@e61c593e9734b47ef462340c24fca8d6a57da14e",
            "phpunit\/phpunit": "9.2.5@ad7cc5ec3ab2597b329880e30442d9054526023b",
            "sebastian\/code-unit": "1.0.3@d650ef9b1fece15ed4d6eaed6e6b469b7b81183a",
            "sebastian\/code-unit-reverse-lookup": "2.0.1@c771130f0e8669104a4320b7101a81c2cc2963ef",
            "sebastian\/comparator": "4.0.2@266d85ef789da8c41f06af4093c43e9798af2784",
            "sebastian\/diff": "4.0.1@3e523c576f29dacecff309f35e4cc5a5c168e78a",
            "sebastian\/environment": "5.1.1@16eb0fa43e29c33d7f2117ed23072e26fc5ab34e",
            "sebastian\/exporter": "4.0.1@d12fbca85da932d01d941b59e4b71a0d559db091",
            "sebastian\/global-state": "4.0.0@bdb1e7c79e592b8c82cb1699be3c8743119b8a72",
            "sebastian\/object-enumerator": "4.0.1@15f319d67c49fc55ebcdbffb3377433125588455",
            "sebastian\/object-reflector": "2.0.1@14e04b3c25b821cc0702d4837803fe497680b062",
            "sebastian\/recursion-context": "4.0.1@a32789e5f0157c10cf216ce6c5136db12a12b847",
            "sebastian\/resource-operations": "3.0.1@71421c1745788de4facae1b79af923650bd3ec15",
            "sebastian\/type": "2.1.0@bad49207c6f854e7a25cef0ea948ac8ebe3ef9d8",
            "sebastian\/version": "3.0.0@0411bde656dce64202b39c2f4473993a9081d39e",
            "symfony\/polyfill-ctype": "v1.17.1@2edd75b8b35d62fd3eeabba73b26b8f1f60ce13d",
            "theseer\/tokenizer": "1.1.3@11336f6f84e16a720dae9d8e6ed5019efa85a0f9",
            "webmozart\/assert": "1.9.0@9dc4f203e36f2b486149058bade43c851dd97451",
            "pocketmine\/pocketmine-mp": "3.14.0@461bc94236ba1f3504b6f5a20eba9df80ef9a7bd"
        }
    },
    "plugins": {
        "MultiWorld": {
            "name": "MultiWorld",
            "version": "1.6.0-beta1",
            "authors": [
                "CzechPMDevs",
                "VixikCZ",
                "fuyutsuki",
                "kaliiks"
            ],
            "api": [
                "3.0.0"
            ],
            "enabled": true,
            "depends": [],
            "softDepends": [],
            "main": "czechpmdevs\\multiworld\\MultiWorld",
            "load": "POSTWORLD",
            "website": "https:\/\/github.com\/CzechPMDevs\/MultiWorld"
        },
        "FormAPI": {
            "name": "FormAPI",
            "version": "1.3.0",
            "authors": [
                "jojoe77777"
            ],
            "api": [
                "3.2.0"
            ],
            "enabled": true,
            "depends": [],
            "softDepends": [],
            "main": "jojoe77777\\FormAPI\\FormAPI",
            "load": "POSTWORLD",
            "website": ""
        },
        "DevTools": {
            "name": "DevTools",
            "version": "1.13.5",
            "authors": [
                "PocketMine Team"
            ],
            "api": [
                "3.4.0"
            ],
            "enabled": true,
            "depends": [],
            "softDepends": [],
            "main": "DevTools\\DevTools",
            "load": "STARTUP",
            "website": "https:\/\/github.com\/PocketMine\/DevTools"
        },
        "BravoClan": {
            "name": "BravoClan",
            "version": "0.8",
            "authors": [
                "Itzdvbravo"
            ],
            "api": [
                "3.14.0"
            ],
            "enabled": true,
            "depends": [],
            "softDepends": [
                "ScoreHud"
            ],
            "main": "Itzdvbravo\\BravoClan\\Main",
            "load": "POSTWORLD",
            "website": ""
        },
        "LightHeal": {
            "name": "LightHeal",
            "version": "1.0.0",
            "authors": [
                "EndermanLetsPlay"
            ],
            "api": [
                "3.0.0"
            ],
            "enabled": true,
            "depends": [],
            "softDepends": [],
            "main": "LightHeal\\Main",
            "load": "POSTWORLD",
            "website": ""
        }
    },
    "parameters": [
        "custom.phar",
        "--memory-limit=1024M",
        "--disable-ansi",
        "--no-wizard",
        "--data=\/home\/minecraft\/multicraft\/servers\/server120687",
        "---plugins=\/home\/minecraft\/multicraft\/servers\/server120687\/plugins",
        "--level-name=world",
        "--server-ip=0.0.0.0",
        "--server-port=25628",
        "--max-players=5"
    ],
    "server.properties": "#Properties Config file\n#Sun Apr 5 09:43:03 UTC 2020\nserver-ip=0.0.0.0\nserver-port=25628\nmax-players=5\nlevel-name=world\nmotd=grieferking.de Testserver\nwhite-list=on\nannounce-player-achievements=on\nspawn-protection=16\ngamemode=0\nforce-gamemode=off\nhardcore=off\npvp=on\ndifficulty=2\ngenerator-settings=\nlevel-seed=\nlevel-type=DEFAULT\nenable-query=on\nenable-rcon=off\nrcon.password=******\nauto-save=on\nview-distance=8\nxbox-auth=on\nlanguage=eng",
    "pocketmine.yml": "# Main configuration file for PocketMine-MP\n# These settings are the ones that cannot be included in server.properties\n# Some of these settings are safe, others can break your server if modified incorrectly\n# New settings\/defaults won't appear automatically in this file when upgrading.\n\nsettings:\n  #Whether to send all strings translated to server locale or let the device handle them\n  force-language: false\n  shutdown-message: \"Server closed\"\n  #Allow listing plugins via Query\n  query-plugins: true\n  #Show a console message when a plugin uses deprecated API methods\n  deprecated-verbose: true\n  #Enable plugin and core profiling by default\n  enable-profiling: false\n  #Will only add results when tick measurement is below or equal to given value (default 20)\n  profile-report-trigger: 20\n  #Number of AsyncTask workers.\n  #Used for plugin asynchronous tasks, world generation, compression and web communication.\n  #Set this approximately to your number of cores.\n  #If set to auto, it'll try to detect the number of cores (or use 2)\n  async-workers: auto\n  #Whether to allow running development builds. Dev builds might crash, break your plugins, corrupt your world and more.\n  #It is recommended to avoid using development builds where possible.\n  enable-dev-builds: false\n\nmemory:\n  #Global soft memory limit in megabytes. Set to 0 to disable\n  #This will trigger low-memory-triggers and fire an event to free memory when the usage goes over this\n  global-limit: 0\n\n  #Main thread soft memory limit in megabytes. Set to 0 to disable\n  #This will trigger low-memory-triggers and fire an event to free memory when the usage goes over this\n  main-limit: 0\n\n  #Main thread hard memory limit in megabytes. Set to 0 to disable\n  #This will stop the server when the limit is surpassed\n  main-hard-limit: 1024\n\n  #AsyncWorker threads' hard memory limit in megabytes. Set to 0 to disable\n  #This will crash the task currently executing on the worker if the task exceeds the limit\n  #NOTE: THIS LIMIT APPLIES PER WORKER, NOT TO THE WHOLE PROCESS.\n  async-worker-hard-limit: 256\n\n  #Period in ticks to check memory (default 1 second)\n  check-rate: 20\n\n  #Continue firing low-memory-triggers and event while on low memory\n  continuous-trigger: true\n\n  #Only if memory.continuous-trigger is enabled. Specifies the rate in memory.check-rate steps (default 30 seconds)\n  continuous-trigger-rate: 30\n\n  garbage-collection:\n    #Period in ticks to fire the garbage collector manually (default 30 minutes), set to 0 to disable\n    #This only affects the main thread. Other threads should fire their own collections\n    period: 36000\n\n    #Fire asynchronous tasks to collect garbage from workers\n    collect-async-worker: true\n\n    #Trigger on low memory\n    low-memory-trigger: true\n\n  #Settings controlling memory dump handling.\n  memory-dump:\n    #Dump memory from async workers as well as the main thread. If you have issues with segfaults when dumping memory, disable this setting.\n    dump-async-worker: true\n\n  max-chunks:\n    #Cap maximum render distance per player when low memory is triggered. Set to 0 to disable cap.\n    chunk-radius: 4\n\n    #Do chunk garbage collection on trigger\n    trigger-chunk-collect: true\n\n  world-caches:\n    #Disallow adding to world chunk-packet caches when memory is low\n    disable-chunk-cache: true\n    #Clear world caches when memory is low\n    low-memory-trigger: true\n\n\nnetwork:\n  #Threshold for batching packets, in bytes. Only these packets will be compressed\n  #Set to 0 to compress everything, -1 to disable.\n  batch-threshold: 256\n  #Compression level used when sending batched packets. Higher = more CPU, less bandwidth usage\n  compression-level: 7\n  #Use AsyncTasks for compression. Adds half\/one tick delay, less CPU load on main thread\n  async-compression: false\n  #Experimental, only for Windows. Tries to use UPnP to automatically port forward\n  upnp-forwarding: false\n  #Maximum size in bytes of packets sent over the network (default 1492 bytes). Packets larger than this will be\n  #fragmented or split into smaller parts. Clients can request MTU sizes up to but not more than this number.\n  max-mtu-size: 1492\n\ndebug:\n  #If > 1, it will show debug messages in the console\n  level: 1\n\nplayer:\n  #Choose whether to enable player data saving.\n  save-player-data: true\n  anti-cheat:\n    #If false, will try to prevent speed and noclip cheats. May cause movement issues.\n    allow-movement-cheats: true\n\nlevel-settings:\n  #The default format that levels will use when created\n  default-format: pmanvil\n\nchunk-sending:\n  #To change server normal render distance, change view-distance in server.properties.\n  #Amount of chunks sent to players per tick\n  per-tick: 4\n  #Radius of chunks that need to be sent before spawning the player\n  spawn-radius: 4\n\nchunk-ticking:\n  #Max amount of chunks processed each tick\n  per-tick: 40\n  #Radius of chunks around a player to tick\n  tick-radius: 3\n  light-updates: false\n  clear-tick-list: true\n  #IDs of blocks not to perform random ticking on.\n  disable-block-ticking:\n    #- 2 # grass\n\nchunk-generation:\n  #Max. amount of chunks in the waiting queue to be populated\n  population-queue-size: 8\n\nticks-per:\n  autosave: 6000\n\nauto-report:\n  #Send crash reports for processing\n  enabled: true\n  send-code: true\n  send-settings: true\n  send-phpinfo: false\n  use-https: true\n  host: crash.pmmp.io\n\nanonymous-statistics:\n  #Sends anonymous statistics for data aggregation, plugin usage tracking\n  enabled: false #TODO: re-enable this when we have a new stats host\n  host: stats.pocketmine.net\n\nauto-updater:\n  enabled: true\n  on-update:\n    warn-console: true\n    warn-ops: true\n  #Can be development, alpha, beta or stable.\n  preferred-channel: stable\n  #If using a development version, it will suggest changing the channel\n  suggest-channels: true\n  host: update.pmmp.io\n\ntimings:\n  #Choose the host to use for viewing your timings results.\n  host: timings.pmmp.io\n\nconsole:\n  #Choose whether to enable server stats reporting on the console title.\n  #NOTE: The title ticker will be disabled regardless if console colours are not enabled.\n  title-tick: true\n\naliases:\n  #Examples:\n  #showtheversion: version\n  #savestop: [save-all, stop]\n\nworlds:\n  #These settings will override the generator set in server.properties and allows loading multiple levels\n  #Example:\n  #world:\n  # seed: 404\n  # generator: FLAT:2;7,59x1,3x3,2;1;decoration(treecount=80 grasscount=45)\n\nplugins:\n  #Setting this to true will cause the legacy structure to be used where plugin data is placed inside the --plugins dir.\n  #False will place plugin data under plugin_data under --data.\n  #This option exists for backwards compatibility with existing installations.\n  legacy-data-dir: false\n",
    "extensions": {
        "Core": "7.3.14",
        "date": "7.3.14",
        "libxml": "7.3.14",
        "openssl": "7.3.14",
        "pcre": "7.3.14",
        "sqlite3": "7.3.14",
        "zlib": "7.3.14",
        "bcmath": "7.3.14",
        "calendar": "7.3.14",
        "crypto": "0.3.1",
        "ctype": "7.3.14",
        "curl": "7.3.14",
        "dom": "20031129",
        "json": "1.7.0",
        "SPL": "7.3.14",
        "ds": "1.2.9",
        "filter": "7.3.14",
        "ftp": "7.3.14",
        "gd": "7.3.14",
        "gmp": "7.3.14",
        "hash": "7.3.14",
        "standard": "7.3.14",
        "leveldb": "0.2.1",
        "mbstring": "7.3.14",
        "mysqlnd": "mysqlnd 5.0.12-dev - 20150407 - $Id: 7cc7cc96e675f6d72e5cf0f267f48e167c2abb23 $",
        "mysqli": "7.3.14",
        "pcntl": "7.3.14",
        "PDO": "7.3.14",
        "pdo_mysql": "7.3.14",
        "pdo_sqlite": "7.3.14",
        "Phar": "7.3.14",
        "pocketmine_chunkutils": "0.1.0",
        "posix": "7.3.14",
        "pthreads": "3.2.1dev",
        "recursionguard": "0.1.0",
        "Reflection": "7.3.14",
        "shmop": "7.3.14",
        "SimpleXML": "7.3.14",
        "sockets": "7.3.14",
        "igbinary": "3.1.2",
        "tokenizer": "7.3.14",
        "xml": "7.3.14",
        "xmlreader": "7.3.14",
        "xmlwriter": "7.3.14",
        "yaml": "2.0.4",
        "zip": "1.15.4"
    }
}
```

**Comment by irpanXstrome** [comment]
Download The MultyWorld Release

**Comment by dktapps** [comment]
This has been fixed on 4.0. I'm not planning to fix this on PM3 since it's on life support now.
